server/controller/routes/getQuestions.js

```diff
@@ -0,0 +1,38 @@
+const getQuestions = require('../../db/queries/getQuestions')
+const  splitString = require('../utils/splitString')
+
+module.exports = async (req, res, next) => {
+
+  let newQuestionsArray = []
+  let updatedNames = []
```

Comment:
Same thing here no need for declaration the and then update it 
just put const before you use it

Reply:
Done